### DISCUSSION TITLE
SDK 1.9.0 Privacy attribute

### DISCUSSION THREAD

**Discussion by luispozoWS** [OP-TED/eForms-SDK#847] [Q&A] [answered]
Hello,
We noticed that there was a “privacy” attribute in certain fields for eForms 
![privacy](https://github.com/OP-TED/eForms-SDK/assets/158502029/7a807390-a9d0-4b31-a8b9-861fea7b55a3)

What is this attribute used for for the fields concerned?
Should it be implemented? If yes, how ?

Best regards

**Comment by bertrand-lorentz** [comment, marked as answer]
Please see https://docs.ted.europa.eu/eforms/1.9/fields/index.html#_withheld_publication_mechanism
and https://docs.ted.europa.eu/eforms/1.9/schema/withheld-publication.html
